src/main/java/teammates/common/datatransfer/attributes/FeedbackSessionAttributes.java

```diff
@@ -449,37 +429,19 @@ public int compare(FeedbackSessionAttributes session1, FeedbackSessionAttributes
      * method is called with combined feedback sessions from many courses
      */
     public static void sortFeedbackSessionsByCreationTimeDescending(List<FeedbackSessionAttributes> sessions) {
-        Collections.sort(sessions, new Comparator<FeedbackSessionAttributes>() {
-            @Override
-            public int compare(FeedbackSessionAttributes session1, FeedbackSessionAttributes session2) {
-                int result = session2.createdTime.compareTo(session1.createdTime);
-                if (result == 0) {
-                    if (session1.endTime == null || session2.endTime == null) {
-                        if (session1.endTime == null) {
-                            --result;
-                        }
-                        if (session2.endTime == null) {
-                            ++result;
-                        }
+        sessions.sort(Comparator.comparing((FeedbackSessionAttributes session) -> session.createdTime).reversed()
+                .thenComparing((FeedbackSessionAttributes session1, FeedbackSessionAttributes session2) -> {
+                    if (session1.endTime == null) {
+                        return -1;
+                    } else if (session2.endTime == null) {
+                        return 1;
                     } else {
-                        result = session2.endTime.compareTo(session1.endTime);
+                        return session2.endTime.compareTo(session1.endTime);
                     }
-                }
-
-                if (result == 0) {
-                    result = session2.startTime.compareTo(session1.startTime);
-                }
-                if (result == 0) {
-                    result = session1.courseId.compareTo(session2.courseId);
-                }
-
-                if (result == 0) {
-                    result = session1.feedbackSessionName.compareTo(session2.feedbackSessionName);
-                }
-
-                return result;
-            }
-        });
+                })
+                .thenComparing(session -> session.startTime).reversed()
```

Comment:
It does not look right to put the `reversed` there since it will reverse the entire chain up to this spot, when all you want is reversing the start time comparator only.